src/test/scala/nl/hugo/redbook/Spec.scala

```diff
@@ -2,6 +2,6 @@ package nl.hugo.redbook
 
 import org.scalatest.{ Matchers, WordSpec }
 
-trait Spec extends WordSpec with Matchers {
+class Spec extends WordSpec with Matchers {
```

Comment:
It wouldn't compile without this change. I'll investigate it a bit more when I have the time.